metadata/Cadastre/turn_gps_baselines.md

```diff
@@ -4,38 +4,67 @@ Turn GPS Baselines
 
 ## ID
 
+2e9a22fc-5c30-4447-ad34-ab171cea21f8
+
 ## Brief Summary
 
-## Summary
+Polyline dataset of the distance between each base station in the TURN GPS network.
 
-## Description
+## Summary
 
-This data was created to display and measure the distance between base stations on the TURN GPS network. Baselines are distance only no physical feature.
+This dataset contains polylines displaying the geographical distance between TURN GPS base stations. The baselines represent calculated distances only and are not a physical feature.
 
-This data represents the measured distance between the The Utah Reference Network Global Positioning System (TURN GPS) base stations. It models the current coverage of the network and in some areas extends past the boundary of Utah. Baselines with a distance greater than 70 km will start to decrease your precision.
+## Description
 
 ### What is the dataset?
 
+The [Utah Reference Network](https://gis.utah.gov/products/turn/), also known as TURN GPS, is comprised of over 100 GPS base stations that provide high-precision, real-time location data and corrections. This dataset contains polylines displaying the geographical distance between all available TURN GPS base stations. These baselines model the current coverage of the network and include areas in and around Utah.
+
 ### What is the purpose of the dataset?
 
+This data was created to display and measure the distance between base stations on the TURN GPS network. For more information on TURN GPS or to learn more on how to use the network, please visit [our website](https://turngps-billpay.ugrc.utah.gov/).
+
 ### What does the dataset represent?
 
+Each feature in this dataset contains the distance between two given base stations in miles and kilometers, as well as the status of that part of the network.
+
+O =
+PR =
+R =
+
+<!--- What do these status codes refer to? --->
```

Comment:
According to Sean, these codes are for internal use only. You could note that as such here.